patroni/ha.py

```diff
@@ -94,6 +94,11 @@ def check_mode(self, mode):
         else:
             return self.patroni.config.check_mode(mode)
 
+    def master_stop_timeout(self):
+        """ Master stop timeout """
+        ret = parse_int(self.patroni.config['master_stop_timeout'])
+        return ret if ret and ret > 0 and self.is_synchronous_mode() else None
```

Comment:
```suggestion
        return ret if ret > 0 and self.is_synchronous_mode() else None
```

Reply:
From the config key we can get anything, not necessarily integer. In this case parse_int() returns None.
The old version could deal with it.

Reply:
:+1: